draft-irtf-cfrg-hpke.md

```diff
@@ -1536,32 +1536,32 @@ In the PSK and AuthPSK modes, the PSK MUST have at least 32 bytes of
 entropy and SHOULD be of length `Nh` bytes or longer. Using a PSK longer than
 32 bytes but shorter than `Nh` bytes is permitted.
 
-HPKE is specified to use HKDF as key derivation function. HKDF is not
-designed to slow down dictionary attacks, see {{?RFC5869}}. Thus, HPKE's
+HPKE is specified to use HKDF as its key derivation function. HKDF is not
+designed to slow down dictionary attacks (see {{?RFC5869}}). Thus, HPKE's
 PSK mechanism is not suitable for use with a low-entropy password as the
-PSK: in scenarios in which the adversary knows the KEM shared secret
-`shared_secret` and has access to an oracle that allows to distinguish between
+PSK: In scenarios in which the adversary knows the KEM shared secret
+`shared_secret` and has access to an oracle that distinguishes between
```

Comment:
The oracle itself does not distinguish between good and wrong PSKs, it is the adversary that uses the oracle to do that.

Reply:
Or:
```suggestion
`shared_secret` and has access to an oracle allowing to distinguish between
```

Reply:
Coming back to it, I think the change done by the editors is good. Please excuse the noise.